src/commitment.rs

```diff
@@ -1,20 +1,16 @@
-use crate::constants;
-use crate::point;
 use crate::polynomial;
 use crate::setup;
-use blst;
-// TODO: ripple oblast through rest of codebase
-use oblast::{Fr, P1, P2};
+use oblast;
```

Comment:
I notice you're quite fond of the module-inclusive identifiers like `oblast::Fr`, I prefer it without the module personally but I'm happy to go modular to match your style :grin:

Reply:
i can do either, in this case it is clear from context so no problems dropping it